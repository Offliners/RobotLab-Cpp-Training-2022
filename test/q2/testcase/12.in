489
fOrtH
CONcISE
loving
ORIGIn
PRogressIve
DISTrACT
LIvE
prOgrAM
WHIle
CrAcKDowN
PROnouN
Alien
IgNoRANt
swaLLOw
MatuRitY
hoSpiTal
minIMuM
LACk
9aJESty
cANVAs
papaya
SIX
MATHeMaTICAl
Pound
BAGGAGE
BiNd
a9coun4ant
Park
clocK
goAL
dAIly
eMb9r5aSs
meSS
palAce
PAUse
HunT
naVy
LIBERaL
lEns
advanced
54887CL6
OuTlinE
EArNEST
E-MaiL
LoOk
5664L9E74
trend
kETCHUp
mAyBe
ASsOcIaTiON
CAb
COASTLine
CORPORATION
REFRIGERATOR
Beijing
clAss
GrAMmaticAL
duST
m02694ave
noble
milliOnaiRE
RESEARcHER
trIAL
AttaINmeNt
lord
DOMAIN
PEnsIoN
l1sEr
beSIdEs
TidE
brew
DangeR
INgenUiTy
FAilURE
sOLAR
wander
weave
ForeheAD
SUnN3
REvive
greet
SHEEP
SIGH
LIPStIcK
ASSoCIAte
cleaNeR
pArTner
dump
SaD
tOlL
LInE
PiGment
sHElteR
tanKer
GEoGrAPhy
IVORY
iNTroDUce
coLa
EnDORSE
cANDiDATE
DEPART
NOTIFY
huge
stIngy
aBove
dElIvERy
exoTiC
raLLy
RoACH
cheerfUl
PEAk
OVER
NAP
abseNce
AqUARiUM
huRricane
girl
permIt
IroNIc
InTERPRET
ungrAteful
VAcancY
cloCkWork
enclose
eNdless
DIpLoMaT
seCond
SCENERy
DrAFT
eXHIbITION
CONtEmptIBle
NEW
MadaM
FoREIgn
hymn
IMAGI7A89V7
cOstume
cOAL
indeBted
86399T12
INvENT
c
resoLUtiOn
solDier
scarED
Sum
INGReDIENT
AbIDE
LIGHt
iNS0RU2T21
assurance
decIDe
c6a8
sMog
donE
dISmAy
lAtElY
skETcH
paRAdOXiCALLY
SUBSIDY
Depict
alLEge
elderLy
ThOsE
AROUSE
Cub
ACcusatiOn
PRoVERB
stratEgy
ox
HEAD
P
LIVESTOCK
voyage
HOCKEY
MARKET
caVALRY
cOnverse
audio-visual
natural
cushioN
humorouSly
eitHeR
AMBIGUitY
gRocErY
RUN
sore
me7288075
THinK
cr794T
GUy
suCh
chalLenging
BREaD
rape
SOAP
Ancestor
identiFicaTion
pAInFUL
REcKon
U3E9ulNes0
quotation
tImE
CoURAGE
wheel
SEntIMEnTAL
insPirE
SolVE
leaF
greAsE
SinK
hINT
LUnCH
EmISSION
SPy
BreEd
MOTHER
GLOOMY
grandchilD
imMOrAL
sITUaTE
SuBjECt
BLOC
coNduCtOr
pond
ma0D16
BEEf
iNTensIve
suckLE
ChrISTIANITY
INTrIGUe
PAIn
DISasTrOUS
BLESSING
AWAY
0922i7788
liCk
waRNInG
ConfEsS
noise
demo
INTRIGUInG
lieUtenanT
MISCELLANEOUS
RigorouS
KItCHEN
mEmBErShIp
bud
JoG
COMmERciaL
E9A24A35O4
iLlOgicAl
vacuuM
blAme
FaulT
InduStriOus
DANcER
cROWD
RoAST
CAGe
PIMPLE
PartIAlLY
enthusiAStIC
mAYOR
HAPPENING
NInEteEN
HURT
soBER
suicIDe
cerTaIN
SadDlE
eXTensioN
moNEtARy
PHYSiCAL
rIghTEoUS
coLONIal
tERRIFIED
tRAVELER
DEcREE
li6
cOMmUNal
DULl
pAiNfULLY
sYllabUs
LamB
epigRaM
INSeCUrE
pRAiseWOrThy
s
E50493
u58ent
humbly
HIRE
PROVE
fiu
lEgal
PErFUmE
COMPATIBLY
Friday
eCOnomy
follOW
sPeECH
gravE
bUsinesslike
Sword
wAItrEsS
SEWER
MONSTER
ADMIRAL
END
INTERnATIOnal
ALLoCATiON
hostEss
allow
fieRce
MARATHON
EdUcaTor
Help
ALLIance
BReaThe
tariff
king
BROCHURE
INterpreter
ADVenture
fouR
AcQUiSiTioN
DEbaTE
EXPLICIT
blOoM
LAMA
rejeCT
iNTeRchANgeAblE
harmful
aUTONOMY
finish
staple
GRIEF
g88L
owl
VERdICt
BOLD
Idiot
dive
vERiFY
NoUn
eNCYCloPedIA
b.c.
DiCtaTor
confide
SLeNdeR
tomb
BASE
molecule
Secretary
MEMORIZE
AchE
uNTiL
sufficIenT
c54l
COncESSION
DiSconTeNT
bErry
ReSORt
livelY
reMemBER
CoMpete
catEr
MiLITaRy
60n981766s
POLITICS
AvENGE
Save
KN10
beauty
SANCtION
sTUpID
PERCEnT
DEepen
COLLISION
appROvAl
manHOOD
eDIT
CompENSAtioN
vital
bull
handy
iTALiaN
theoRetIcal
reSIST
grEasy
cOnsUme
urgent
mAinTAIn
CHILDCaRE
MESSAGe
wEekLy
LandmIne
DoUgh
ofFsPrINg
THOUGHTFUL
acQuAiNTance
undergO
villain
teaM
sITe
BROADcAST
sake
MOB
9AT01727
deSpite
CoIncIDENCe
dEsTRUCTIoN
525l
conStiTute
BABY-SITTER
INcuR
none
inTegral
sUPerStITIoN
reSCuE
EITHER
plane
TEXTBOOK
POultRy
MasTErpiECe
damage
PRoTeiN
man
TIE
Doze
SHARE
iDentity
prEfer
DISasTeR
sorRy
ASYLUM
LOWER
chocolate
920923b5E
peNGUIN
BLONDe
DISABLe
stuff
ACCOUNTING
rEpReSENTATIvE
tRaNsLATor
Mild
nOTe
turTle
HANDKErchiEF
CRASH
neighborhood
invoke
VaLlEY
M4t3Ri8L
friGhten
insulting
dI72924t
dEVOTIoN
AUTHoriTY
dIlIGENcE
TROPICAL
ETHIC
hug
iNDEPENDENTLY
nEEDY
Revise
NECKLACE
bAcoN
whose
MuRdERer
M478
FRIENDsHIP
diviNE
conceal
posture
ExPReSS
hyPhen
nEpheW
cONsUlTaTIon
Mall
BuildiNg
rEStRICT
division
pALm
TRoPHY
cERAmIc
WEATHER
