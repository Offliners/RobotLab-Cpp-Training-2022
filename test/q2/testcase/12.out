adj. 迷信的
v. 復活
n. 分類
Unknown
n. 過敏性反應
n. 評論家
v. 承認
n. 世紀
v. 鞏固
n. 懇求
adj. 敵人的
n. 蚯蚓
Unknown
adj/adv./v. 堅決的；使穩固
v. 耕作；培養
n./v. 行動；表演
n./ v. 輕搖；慢跑
n./v. 年齡；使變老
n. 鹿
v. 低估
n. 泥
n. 醋
n. 鱷魚
v. 發生
adj. 諷刺的
n. 畢業；畢業典禮
n. 事故
n. 詩
n. 污物；泥土
adj./v. 滿足的
n. 科學家
n. 熊；魯莽之人
adj. 公共的
n./v. 調查；審視
v. 追捕
n. 歡樂
n./v. 功能；運行
n. 說明文字
n. 戰場
adj. 值得讚揚的
n. 鞋帶；花邊
adj. 真心誠意的
n./v. 星；明星
n. 午餐；午宴
n. 點鐘
n. 髮型
adj. 個人的
n. 布料；衣料
n./v. 基礎
Unknown
n./v. 調動
n. 葡萄柚
v. 發怨言
adj. 動力的；動態的
n. 繁殖者
n. 代名詞
n. 愚蠢的錯誤
n. 灰泥
adj. 祖先的
adj. 硬的
n. 四分之一
n./v. 飛澱(聲)
adv. 舒適地
n./v. 包裹；捆
v. 占領
adj. 可理解的
n. 身分；同一性
n. 調查者
adj. 整潔的
n. 美髮師
v. 修剪；刪除
n. 愛國者
adj. 合適的
n./v. 負擔；加負擔於…
n. 動量；動向
n. 收入；工資
n./v. 群；聚集
v. 相信
n. 反映
adj. 相比而言的
n. 麵粉
n. 很多
n./v. 買賣；便宜貨
n. 八月
adj. 生氣的
adj. adj. 原住民的
n. 長統襪及短襪
n./v. 疼痛
n. 蛋；卵細胞
n. 惡意
n. 場所
v. 鎮壓
Unknown
adj. 勝利的
adj. 經濟的；節儉的
n. 隆隆聲；景氣
n. 選舉
v. (n〕決定
n./v. 許可；執照
n. 欽佩
adj. 對…不感興趣
Unknown
adj. 理論的
n. 小狗
v. 打瞌睡
n./v. 瓶子；把…裝瓶
n. 名人(聲)
v. 強化
n. 演奏會；音樂會
n. 稻草；麥桿
n./v. 圖表
v. 使再循環；使回收使用
n. 先生
n. 領帶
n. 拒絕
v. 不滿
n. 所有的裝備
n. 不幸；悲慘
n.  原著居民
n./v. 開始
v. 談話
n. 電話
n. 根
n. 女管家
n./v. 製造業；製造
n. 獎品
adj. 數學的
n. 修女；尼姑
n. 原野；欄位
n. 好萊塢
prep./adv. 沿著；一起
n. 門
n. 照料小孩
adj. 相容的
Unknown
v. 使傾覆；推翻
v. 彎腰
v. 使滿足
adj. 厚的
n. 冒險(行為)；投機
n. 博物館
n. 航程；飛機
v. 收集；聚集
n./v. 停留
n. 命運
n./adj. 胚胎；未成熟的
adj. 美麗的
v. 使銘記
n. 服務員
n. 賜福；同意
n. 節日；歡樂
n. 瓷器
v. 逗留
n. 郡；縣
adj. 奇怪的
n./v. 看
adj. 公正的
v. 懷疑
v. 同意；認可
n. 惡性腫瘤；癌症
n. 康復
n./v. 擺渡；渡輸
n. 角度；立場
n. 會議
adj. 有尖角的
n. 天使
n./v. 契約
n. 公雞
v. 趕走
prep./adv. 在遠處
adv./conj. 然而
n. 爭吵
n. 階級組織
adj. 結構上的
v. 依賴
v. 視為
v. 侵蝕
n. 誘惑
n. 百分之…
adj. 便宜的
Unknown
adj. 脆弱的
n. 公司
n. 硬幣
n. 悲傷；不幸
adj. 完全的；絕對的
n. 金魚
adj. 寂靜的
n. 陽臺；樓座
n. 為了…的利益；代表
n. 椰子
Unknown
n. 評論家
n. 奢侈
adv. 不知怎麼地
adj. 令人憂心的
n. 回聲；回響
v. 使有尊嚴
n. 遊戲；競賽
adj. 聰明的
adj./prep. 過去的
n./v. 嘆息
n. 毛線衫
n. 金屬
n. 人性；人文學科
v./n. 窺視；偷看
n./V. 私語
v. 發現
Unknown
n. (被)起訴
adj./pron. (兩者之中)任一的
v. 使重疊
v. 確保
v. 哭泣
n. 緊急情況
Unknown
n. 十二月
adv. 永遠
n. 喪失；遺失
adj. 法律上的
n. 路
n. 法令；條例
adj./n./v. 平均的
adj./n. 塑膠
adj. 嚴格的
v. 捐獻；貢獻
adj. 武裝的
n. 斧頭
n. 害蟲
adv. 法律上
prep/adv. 在外
n. 部門；系
n. 流行
adj. 有毒的
n. 四月
n. 退款
v. 歸納
n./v. 封面；遮蓋
adj./adv. 分開的
n. 起重機；鶴
adj. 多產的
n. 陸軍中尉
v. 欺騙
n. 鄰居
n. 軍隊；團體
n./v. 陳列
adj. 慣例的；協定上的
v. 跛行
n. 教育工作者
prep./adv. 在…之上
v. 敵發
adj. 不幸的
n. 女同性戀者
adj. 更動
Unknown
v. 發射
n. 宣稱
adj. 輕快的
n./v. 罷工；猛然攻擊
n. 主管；導演
prep./adv. 在…後面；落後
n. 垃圾；廢物
n. 計算
n. 村莊
n. 烏鴉；雞啼聲
